src/app/shared/services/api/archive.repo.ts

```diff
@@ -153,6 +153,10 @@ export class ArchiveRepo extends BaseRepo {
 
     return this.http.sendRequestPromise<ArchiveResponse>('/profile_item/delete', data, ArchiveResponse);
   }
+
+  public getArchiveTags(archiveId: string) {
```

Comment:
Since this is specifically for getting public tags, can we change this method name to something like `getPublicArchiveTags` or something similar so the name isn't confusing?

Reply:
@meisekimiu I have renamed the method according to your suggestion